### PR TITLE
Updated sample to Angular v 14

### PR DESCRIPTION
**Pull request by salman90**
## Purpose

* Updated  the following:
1. Angular version.
2. MSAL browser and MSAL Angular version.
3. Configuration script.
  
Looking forward for your feedback. 

## Does this introduce a breaking change?

```
[ x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ x] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -19,12 +19,11 @@ export class HomeComponent implements OnInit {
         filter((msg: EventMessage) => msg.eventType === EventType.LOGIN_SUCCESS),
       )
       .subscribe((result: EventMessage) => {
-        console.log(result);
         const payload = result.payload as AuthenticationResult;
         this.authService.instance.setActiveAccount(payload.account);
+        this.setLoginDisplay();
```
Comment (derisen) [inline src/app/home/home.component.ts:24]:
```suggestion
        this.setLoginDisplay();
```
do you need to call setLoginDisplay() here?

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -19,12 +19,11 @@ export class HomeComponent implements OnInit {
...
       });
-
-      this.setLoginDisplay();
+       this.setLoginDisplay();
```
Comment (derisen) [inline src/app/home/home.component.ts]:
```suggestion
       this.setLoginDisplay();
```
can you check the indentation here?

File: src/app/profile/profile.component.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ type ProfileType = {
   styleUrls: ['./profile.component.css']
 })
 export class ProfileComponent implements OnInit {
-  profile!: ProfileType;
+  profile!: ProfileType | null;
```
Comment (derisen) [inline src/app/profile/profile.component.ts]: would this ever be null?

File: src/app/home/home.component.ts (hunk 1 above)
Comment (salman90) [reply src/app/home/home.component.ts:24]: Yes we do, After `LOGIN_SUCCESS` the component doesn't update, so we need to call  `setLoginDisplay` again after `LOGIN_SUCCESS`.

File: src/app/home/home.component.ts (hunk 2 above)
Comment (salman90) [reply src/app/home/home.component.ts]: Done.

File: src/app/profile/profile.component.ts (hunk 3 above)
Comment (salman90) [reply src/app/profile/profile.component.ts]: Done

**Review by derisen — APPROVED** [review]
LGTM, thanks @salman90

**Pull request merged**